tests/cli.js

```diff
@@ -143,6 +143,48 @@ describe('cli', function () {
     });
   });
 
+  // Test default config files
+
+  it('should return JSON from .sass-lint.yml', function (done) {
+    var command = 'sass-lint ../../sass/cli.scss --verbose';
+
+    childProcess.exec(command, { cwd: path.join(__dirname, 'yml', '.sass-lint.yml') }, function (err, stdout) {
```

Comment:
I changed to `cwd` of each process to test the feature. I didn't know exactly where to put those files so I create 2 dedicated directories for them.